BruTile/Web/ArcGisHttpTileSource.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) BruTile developers team. All rights reserved. See License.txt in the project root for license information.
+
+// This file was created by Tim Ebben (Geodan) 2009
+
+using BruTile.Cache;
+
+namespace BruTile.Web;
+
+public class ArcGisHttpTileSource(string baseUrl, ITileSchema schema, IPersistentCache<byte[]> persistentCache = null)
```

Comment:
The ArcGisHttpTileSource could also be an ordinary tile source, but it is not used an may need to be removed. I have to check if you use it in Mapsui.